washboard-ui/packages/washboard-ui/src/app/components/lattice-settings.tsx

```diff
@@ -20,36 +20,48 @@ const formSchema = z.object({
       },
       {message: 'Could not connect to Lattice'},
     ),
+  latticeId: z.string(),
+  ctlTopicPrefix: z.string(),
+  retryCount: z.number().min(0),
 });
 
 type LatticeFormInput = z.input<typeof formSchema>;
 
 type LatticeFormOutput = z.output<typeof formSchema>;
 
 export function LatticeSettings(): ReactElement {
-  const [{latticeUrl}, setConfig] = useLatticeConfig();
+  const [{latticeUrl, latticeId, ctlTopicPrefix, retryCount}, setConfig] = useLatticeConfig();
   const form = useForm<LatticeFormInput, object, LatticeFormOutput>({
     resolver: zodResolver(formSchema),
     defaultValues: {
-      latticeUrl: latticeUrl,
+      latticeUrl,
+      latticeId,
+      ctlTopicPrefix,
+      retryCount,
     },
   });
 
-  function onSubmit(data: LatticeFormOutput): void {
+  function onSubmit({latticeUrl, latticeId, ctlTopicPrefix, retryCount}: LatticeFormOutput): void {
     setConfig({
-      latticeUrl: data.latticeUrl,
+      latticeUrl,
+      latticeId,
+      ctlTopicPrefix,
+      retryCount,
     });
   }
 
   useEffect(() => {
     form.setValue('latticeUrl', latticeUrl);
+    form.setValue('latticeId', latticeId);
+    form.setValue('ctlTopicPrefix', ctlTopicPrefix);
+    form.setValue('retryCount', retryCount);
   }, [form, latticeUrl]);
```

Comment:
```suggestion
  }, [form, latticeUrl, ctlTopicPrefix, latticeId, retryCount]);
```
See lint warning

Reply:
Ahh thanks for the catch -- made the change, also added a `lint` target that does `lint:eslint`!